README.md

```diff
@@ -132,6 +132,6 @@ The New Relic Ruby agent also uses source code from third-party libraries. Full
 
 Thank you, and may your application scale to infinity plus one.
 
-Lew Cirne, Founder
+New Relic Ruby agent team
```

Comment:
👍 This change has already been made across all other agents and is appreciated. Would you please also change the "Thank you" quote from the line immediately above this one? It can say simply "Thank you" or some other cool quote that we could attribute to the Ruby agent team.

Reply:
Thanks, James! I looked through and made all the changes you suggested.